ci/scale-pure-ovn-hosts.sh

```diff
@@ -23,8 +23,10 @@ $OVNSUDO docker images
 
 # Deploy the containers
 pushd $OVN_SCALE_TOP
-$OVNSUDO /usr/local/bin/ansible-playbook -i $OVN_DOCKER_HOSTS ansible/site-ovn-only.yml -e @$OVN_DOCKER_VARS  -e enable_rally_ovs="no" \
-     --extra-vars "ovs_repo=$OVS_REPO" --extra-vars "ovs_branch=$OVS_BRANCH" --extra-vars "configflags=$CONFIG_FLAGS" -e action=deploy
+$OVNSUDO /usr/local/bin/ansible-playbook -i $OVN_DOCKER_HOSTS ansible/site-ovn-only.yml \
```

Comment:
How about adding a comment above this (something like this):

# TODO(flaviof): When this is hooked into a CI system to test a specific OVS commit sha, we will need
# to add back support for passing in the OVS_REPO, OVS_BRANCH, and CONFIG_FLAGS variables.